packages/fuel-indexer-database/postgres/src/lib.rs

```diff
@@ -612,6 +612,25 @@ pub async fn index_id_for(
     Ok(id)
 }
 
+pub async fn penultimate_index_id_for(
+    conn: &mut PoolConnection<Postgres>,
+    namespace: &str,
+    identifier: &str,
+) -> sqlx::Result<i64> {
+    #[cfg(feature = "metrics")]
+    METRICS.db.postgres.penultimate_index_id_for_calls.inc();
+
+    let query = format!(
+    "SELECT id FROM index_registry WHERE namespace = '{namespace}' AND identifier = '{identifier}' ORDER BY id DESC LIMIT 1, 1",
+    );
```

Comment:
Interestingly, I get this error message whenever I do the revert operation:
```
ERROR fuel_indexer::service: 328: Failed to find Index(fuel_examples.explorer_index): error returned from database: LIMIT #,# syntax is not supported
```

Reply:
hmm interesting, I haven't seen that error on my side. Did you run migrations before testing? What postgres version are you on?

Reply:
PostgreSQL 14, the one that's installed through `brew install postgresql`. And yep, I've run the migrations fresh on each test.

It looks like you can use `LIMIT X OFFSET Y` to skip an arbitrary number of rows: https://www.postgresql.org/docs/current/queries-limit.html; testing it locally worked for me.